skins/shabushabu.scss

```
@@ -0,0 +1,16 @@
+.amp {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.shabushabu {
```